lib/ice_cube/builders/string_builder.rb

```diff
@@ -13,14 +13,18 @@ def piece(type, prefix = nil, suffix = nil)
     end
 
     def to_s
-      @types.each_with_object(@base || '') do |(type, segments), str|
+      string = @base || ''
+      @types.each do |type, segments|
```

Comment:
Sounds good to me!!!

On Tue, Mar 17, 2015 at 11:00 AM, Isaac Seymour <notifications@github.com>
wrote:

> In lib/ice_cube/builders/string_builder.rb
> <https://github.com/seejohnrun/ice_cube/pull/273#discussion_r26560481>:
>
> > @@ -13,14 +13,18 @@ def piece(type, prefix = nil, suffix = nil)
> >      end
> >
> >      def to_s
> > -      @types.each_with_object(@base || '') do |(type, segments), str|
> > +      string = @base || ''
> > +      @types.each do |type, segments|
>
> Would love to but I don't have push rights on your fork :P I'll add them
> to gocardless/ice_cube (along with French translation) and PR them into
> your fork.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/seejohnrun/ice_cube/pull/273/files#r26560481>.
>



-- 
David Gil Pérez
qoolife.com
tel: +34 637 896 543 | skype: dgilperez | @dgilperez
<http://twitter.com/dgilperez>